.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md

```diff
@@ -23,3 +23,7 @@ assignees: ''
 ## Screenshot
 
 [if relevant, include a screenshot]
+
+## Browser details
+
+[optionally - if you can, copy the report generated by [mybrowser.fyi](https://mybrowser.fyi/) - this might help us debug certainly types of issues]
```

Comment:
LGTM, can you capitalise "optionally" please.

Reply:
none of the other text is capitalised... do you want me to change all of them?